public/pages/Overview/utils/helpers.ts

```diff
@@ -121,6 +123,21 @@ export function getVisualizationSpec(description: string, data: any, layers: any
   };
 }
 
+export const getYDomainRange = (data: any[], timeUnit: string): number[] => {
+  data = data.filter((item) => item.finding === 1);
+
+  let dateFormat = 'mm';
+  const timeUnitSize = timeUnit.match(/.*(seconds|minutes|hours|date|month|year)$/);
+  if (timeUnitSize) {
+    if (timeUnitSize && timeUnitSize[1]) {
+      dateFormat = `${timeUnitSize[1][0]}${timeUnitSize[1][0]}`;
```

Comment:
`${timeUnitSize[1][0]}${timeUnitSize[1][0]}` -- are both the indices supposed to be same?

Reply:
Yes, these format goes like `hh` or `mm`.